CHANGELOG.md

```diff
@@ -7,11 +7,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
-## [0.47.0] - 2024-08-05
+## [0.47.0] - 2024-08-28
 
 ### Breaking changes
 
--   Updated the prebuilt UI designs
+The value of the following palette variables have changed
```

Comment:
- Please also add a short summary (something like: Reworked the design of the pre-built ui and moved to a monochrome design.)
- Add a `#### Details`  between the above summary and the detailed changelog
- Make the sub-headings (Palette changes, Provider button style changes, Icon updates) into actual markdown subheadings.

Reply:
Done